bin/export.sh

```diff
@@ -54,15 +65,18 @@ WORKDIR=$(mktemp -d)
 ##
 (
     cd ${SRC}
-    TAG=$(git describe --tags --always)
     MTIME=$(date -d @`git log -1 --pretty=format:%ct tags/${TAG}` '+%Y-%m-%d %H:%M:%S')
     ARCHIVE=$DIR/qpid-proton-${TAG}.tar.gz
+    VERSION=$(git show tags/${TAG}:version.txt)
+    PREFIX=qpid-proton-${VERSION}
     [ -d ${WORKDIR} ] || mkdir -p ${WORKDIR}
-    git archive --format=tar --prefix=qpid-proton-${TAG}/ tags/${TAG} \
+    git archive --format=tar --prefix=${PREFIX}/ tags/${TAG} \
         | tar -x -C ${WORKDIR}
+    ${SRC}/bin/version.sh ${WORKDIR}/${PREFIX} ${VERSION}
```

Comment:
This doesn't make sense to me. This script is supposed to simply export exactly the contents of a tag in the git repo. This tag is generally a signed release, and running scripts over the content of the release will invalidate that process entirely.